### PR TITLE
Used extended base container for Fusioncatcher

### PR DESCRIPTION
**Pull request by Redmar-van-den-Berg**
This should fix [issue-20337](https://github.com/bioconda/bioconda-recipes/issues/20337)

**Replace this sentence with a detailed description of your pull request**

----

> Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences**
      (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

> Everyone has access to the following BiocondaBot commands, which can be given in a comment:
>
>  * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
>  * `@BiocondaBot please add label` will add the `please review & merge` label.
>  * `@BiocondaBot please fetch artifacts` will post links to packages and docker containers built by the CI system. You can use this to test packages locally before merging.
>
> For members of the Bioconda project, the following command is also available:
>
>  * `@BiocondaBot please merge` will cause packages/containers to be uploaded and a PR merged. Someone must approve a PR first! This has the benefit of not wasting CI build time required by manually merging PRs.
>
> If you have questions, please post them in gitter or ping `@bioconda/core` in a comment (if you are not able to directly ping `@bioconda/core` then the bot will repost your comment and enable pinging).

### CODE REVIEW

**Comment by npavlovikj** [comment]
@Redmar-van-den-Berg , the extended image **does not** contain the GNU tools you need. `paste` and `sort` are in the `coreutils` packages, while `grep` is its own package. Please add those two dependences in the `run` section`.

File: recipes/fusioncatcher/meta.yaml [hunk 1]
```diff
@@ -60,3 +60,6 @@ extra:
   identifiers:
     - doi:10.1101/011650
   notes: download-human-db.sh should be updated when new version of FusionCatcher is released.
+  container:
```
Comment (npavlovikj) [inline recipes/fusioncatcher/meta.yaml]: Please remove the whole `container` section.

**Comment by npavlovikj** [comment]
@BiocondaBot please fetch artifacts

**Comment by npavlovikj** [comment]
@BiocondaBot please merge

**Comment by npavlovikj** [comment]
@Redmar-van-den-Berg , `fusioncatcher` is now updated to include the needed GNU tools, https://anaconda.org/bioconda/fusioncatcher/files. Can you please give it a try, and see if it works?

**Comment by Redmar-van-den-Berg** [comment]
@npavlovikj, thanks for including the needed dependencies, I did not know they could be installed on top of the default container. I've tested the new image and it runs without error. Thanks!

**Pull request merged**